### PR TITLE
[hotfix][docs] Fulfill some empty description in Config Doc

### PR DESCRIPTION
**Pull request by KarmaGYZ**
## What is the purpose of the change

This PR fulfill some empty description in Config Doc and relevance javadoc.

## Brief change log

Fulfill empty description in
  - common section
  - core configuration section
  - history server configuration
  - job manager configuration
  - metric configuration
  - resource manager configuration
  - task manager configuration
  - web configuration

javadoc in ResourceManagerOptions.java

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts(all no):

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? no

### CODE REVIEW

File: flink-core/src/main/java/org/apache/flink/configuration/WebOptions.java [hunk 1]
```diff
@@ -89,78 +95,89 @@
...
 
 	/**
 	 * The log file location (may be in /log for standalone but under log directory when using YARN).
 	 */
 	public static final ConfigOption<String> LOG_PATH =
 		key("web.log.path")
 			.noDefaultValue()
-			.withDeprecatedKeys("jobmanager.web.log.path");
+			.withDeprecatedKeys("jobmanager.web.log.path")
+			.withDescription("Define the log file location (may be in /log for standalone but under log directory when using YARN).");
```
Comment (zentol) [inline flink-core/src/main/java/org/apache/flink/configuration/WebOptions.java]: let's get some consistency in here. Some descriptions start with `The config parameter`, others with `Config parameter defining`, and others with `Define`.

I know you just copied the javadocs, but imo this isn't sufficient for the docs.

Comment (KarmaGYZ) [reply flink-core/src/main/java/org/apache/flink/configuration/WebOptions.java]: Thanks for the advise.

Comment (zentol) [reply flink-core/src/main/java/org/apache/flink/configuration/WebOptions.java]: Some examples:
`parallelism.default`: "Default parallelism for jobs."
`metrics.scope.delimiter`: "Delimiter used to assemble the metric identifier."

Basically, remove repetitive prefixes that could be used for every description.

**Comment by KarmaGYZ** [comment]
@zentol Thanks for the concise and clear example.
I push a new commit include:
- Remove repetitive prefixes
- Make some supplement for time unit

For the repetitive prefixes, should we remove all these prefixes in this documentation?

**Comment by zentol** [comment]
Let's stick to the missing ones for now, I don't think anyone has time to review such a cleanup.

File: flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java [hunk 2]
```diff
@@ -135,14 +139,18 @@
...
 	/**
 	 * Interval between probing of system resource metrics specified in milliseconds. Has an effect only when
 	 * {@link #SYSTEM_RESOURCE_METRICS} is enabled.
 	 */
 	public static final ConfigOption<Long> SYSTEM_RESOURCE_METRICS_PROBING_INTERVAL =
 		key("metrics.system-resource-probing-interval")
-			.defaultValue(5000L);
+			.defaultValue(5000L)
+			.withDescription("Interval between probing of system resource metrics specified in milliseconds. Has an effect" +
+				" only when SYSTEM_RESOURCE_METRICS is enabled");
```
Comment (zentol) [inline flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java]: please refer to the actual configuration key for `SYSTEM_RESOURCE_METRICS ` instead, i.e. `"Interval ... only when '" + SYSTEM_RESOURCE_METRICS.key() + "' is enabled."`.

The description is also missing a final period.

**Pull request merged**